README.md

```diff
@@ -1,2 +1,133 @@
-# to-do-list-website
-"To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. 
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [To-do-list] <a name="about-project"></a>
+
+**[To-do-list]** is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client </summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript#">JavaScript</a></li>
+    <li><a href="https://webpack.js.org">Webpack</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[built using webpack]**
+- **[interactive list]**
+- **[Add and remove items]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://VaheStepanyan100.github.io/to-do-list-website)
```

Comment:
The link to your live demo is broken🥲. Kindly add the right link to the deployed app or take the section away as it is not required in this milestone since we are not required to deploy this project😉 as it is giving your readme a less professional look👍.

![image](https://user-images.githubusercontent.com/64643995/236816899-64e6d92b-18f5-48e9-82bc-fc159c6be2da.png)